code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -157,6 +158,7 @@
 	var/secondary_magazine_type
 	///the secondary magazine
 	var/obj/item/ammo_box/magazine/secondary_magazine
+	projectile_damage_multiplier = 1.2
```

Comment:
move this above all of the new variable declarations

Reply:
I'm going to blame me being unable to understand this on it being 2:30AM but by new variable declarations you mean everything below the inhand dimensions vars?

Reply:
Uh, it should be good now?